cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -62,6 +62,14 @@
   </property>
 
   <property>
+    <name>local.kafka.dir</name>
+    <value>libkafka</value>
```

Comment:
Since this is only used by cdap-standalone, I suggest to put it in cdap-site.xml under the cdap-standalone source, otherwise, this option is visible for the cluster mode, which doesn't really make sense. Also, we kind of have all extension jars put under the `ext` directory (see security and core for example).